### ISSUE TITLE
no display when running pretrained model v2 on ubuntu

### ISSUE THREAD

**Issue by kardSIM** [PWhiddy/PokemonRedExperiments#202]
I tried to run the run_pretrained_interactive.py 
everything seems to work correctly 

(pokey3) youssef@youssef-IdeaPad-Gaming-3-15IMH05:~/Desktop/PokemonRedExperiments/v2$ python run_pretrained_interactive.py 
UserWarning: Using SDL2 binaries from pysdl2-dll 2.30.2
using checkpoint: runs/poke_62914560_steps.zip, which is 223.45079647998014 hours old

loading checkpoint
Wrapping the env with a `Monitor` wrapper
Wrapping the env in a DummyVecEnv.
Wrapping the env in a VecTransposeImage.

![Image](https://github.com/user-attachments/assets/b8187c7f-0d0a-4b7e-98e3-fb4de0719a42)

but the emulation is not displayed
am i the only one facing this issue

**Comment by PWhiddy** [comment]
hey! 
have you modified any scripts? 
Does pyboy open up a window if you just invoke it directly via `python -m pyboy ../PokemonRed.gb`?

**Comment by kardSIM** [comment]
i didnt change anything in the code to make sure i cloned the project again 
running python -m pyboy ../PokemonRed.gb result in same issue 

![Image](https://github.com/user-attachments/assets/01c2b52d-b52d-4552-be44-2fae6d55820f)

**Comment by kardSIM** [comment]
Hi again  after installing 
conda install -c conda-forge sdl2_ttf

it solved the issue for me 
thank you

**Comment by kardSIM** [comment]
@PWhiddy  is it possible to change the reward policy and fine tune the pretrained model instead of training from scratch ?

**Comment by PWhiddy** [comment]
Glad you were able to get pyboy working! 

And yes it is possible to make changes and fine tune, although it may or may not work well. If the changes are too abrupt it could cause the policy to collapse. The v2 training script already has code for restarting a checkpoint, and see `go_forever.sh` for how to pipe in a checkpoint file for it to start from!
